actionpack/lib/abstract_controller/base.rb

```diff
@@ -78,7 +78,10 @@ def action_methods
             # Except for public instance methods of Base and its ancestors
             internal_methods +
             # Be sure to include shadowed public instance methods of this class
-            public_instance_methods(false)).uniq.map(&:to_s)
+            public_instance_methods(false))
+
+          methods.uniq!
+          methods.map!(&:to_s)
```

Comment:
Not sure, but I think we can remove `uniq!`,  since the execution of `to_set` below should remove duplicate elements?

Reply:
Right you are!

```
irb(main):002:0> ["foo", "foo"].to_set
=> #<Set: {"foo"}>
```

Send me a PR? Or I can push to master.

Reply:
#33693